# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 7e54a223001d
# Full Hash: 7e54a223001dd1c0057b74c7691dc3821947a3df
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 3
# Description:
#   Bug 1742437 - move DOM script context into own file and rename it; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138132
# ==============================================================================

diff -r 3a34d557d474 -r 7e54a223001d dom/script/moz.build
--- a/dom/script/moz.build	Thu Feb 17 15:57:51 2022 +0000
+++ b/dom/script/moz.build	Thu Feb 17 15:57:51 2022 +0000
@@ -26,6 +26,7 @@
     "ScriptDecoding.h",
     "ScriptElement.h",
     "ScriptKind.h",
+    "ScriptLoadContext.h",
     "ScriptLoader.h",
     "ScriptLoadRequest.h",
     "ScriptSettings.h",
@@ -40,6 +41,7 @@
     "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
     "ScriptElement.cpp",
+    "ScriptLoadContext.cpp",
     "ScriptLoader.cpp",
     "ScriptLoadHandler.cpp",
     "ScriptLoadRequest.cpp",
